### PR TITLE
Fortigate - remove prefixes and parse more fields

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**
https://github.com/SekoiaLab/integration/issues/1095

1. As [FortinetFortiGate and FTNTFGT](https://community.fortinet.com/t5/FortiGate/Technical-Tip-FortiGate-adding-FTNTFGT-prefix-while-sending-logs/ta-p/408432) are just default prefixes, I believe it would make sense to remove them  to avoid confusing and repetition while parsing the very same fields.
2. dstuser [is defined](https://docs.fortinet.com/document/fortigate/7.0.0/new-features/900457/add-dstuser-field-to-utm-logs-7-0-2) as "the username of a destination device when that user has been authenticated on the FortiGate.", so it would seem it should go to the `destination.user.name`. But we already have `duser` there, present in the event as well. 
It seems that `duser` should represent the user authorized on a client machine, while `dstuser` - on the server. So for the case when we have `dstuser` value, parser would assign it to the `destination.user.name` and `duser` to `source.user.name`

## Type of Change

- [ ] New intake format
- [X] Update to existing intake format
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product:

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`poetry run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`poetry run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Streamline the Fortinet FortiGate parser by removing legacy FTNTFGT and FortinetFortiGate prefixes from CEF messages and field extractions, simplify date and field mapping steps, and introduce a new application list field with accompanying test coverage.

New Features:
- Strip FTNTFGT and FortinetFortiGate prefixes from CEF input and field mappings
- Introduce fortinet.fortigate.applist field

Enhancements:
- Remove redundant prefix‐based date parsing steps
- Simplify parser field mappings by dropping legacy prefixed fallback keys
- Consolidate CEF parsing logic into a single prefix‐stripping step

Tests:
- Add new traffic_forward.CEF-6.json test case to cover updated parsing behavior

### CODE REVIEW

File: Fortinet/fortigate/tests/attack_client_side.json [hunk 1]
```diff
@@ -17,12 +17,11 @@
...
-      "type": "ips"
+      "type": "signature - ips"
     },
     "destination": {
-      "address": "5.6.7.8",
-      "domain": "5.6.7.8",
-      "ip": "5.6.7.8",
+      "address": "example.com",
+      "domain": "example.com",
       "port": 80
```
Comment (mchupeau-sk) [inline Fortinet/fortigate/tests/attack_client_side.json]: ip is lost
